src/3d/qgstessellatedpolygongeometry.h

```diff
@@ -46,6 +46,11 @@ class QgsTessellatedPolygonGeometry : public Qt3DRender::QGeometry
     //! Sets whether the normals of triangles will be inverted (useful for fixing clockwise / counter-clockwise face vertex orders)
     void setInvertNormals( bool invert ) { mInvertNormals = invert; }
 
+    //! Returns whether also triangles facing the other side will be created. Useful if input data have inconsistent order of vertices
```

Comment:
\since QGIS 3.2

Reply:
Shall we use that also for classes that are not exported?

Reply:
Well this class itself has a since annotation, so the completionist in me says "yes"